app/javascript/packs/pages/admin/custom_layouts.js

```diff
@@ -0,0 +1,97 @@
+import log from "../../utils/log";
```

Comment:
What can we do to make a `BinxAdminImageUploader` class which is shared by blogs and custom layouts, so that we don't need all this nearly identical code?

Reply:
I can make this happen. Would you rather I used some conditionals for the template, or just use css to hide some of the things I'm not including in this one?

Reply:
I think css will be better, because it's simpler and will keep the js cleaner.